package.json

```diff
@@ -41,6 +40,7 @@
         "babel-minify-webpack-plugin": "0.3.1",
         "compression": "1.7.3",
         "file-loader": "1.1.11",
+        "hadouken-js-adapter": "^1.43.1",
```

Comment:
are you sure that the caret is necessary?  Sometimes npm adds these.  But I don't know if it's necessary here.

Reply:
its okay unless we want to lock versions which is usually what I like to do. I just wasn't told that we should lock versions, I talked about it once and somebody else took care of this.

Reply:
@carterbs Should we lock package versions as soon as we install them?